modules/ingester/instance.go

```diff
@@ -478,6 +486,13 @@ func (i *instance) resetHeadBlock() error {
 	return nil
 }
 
+func (i *instance) getDedicatedColumns() []backend.DedicatedColumn {
+	if cols := i.overrides.DedicatedColumns(i.instanceID); len(cols) > 0 {
```

Comment:
If it should be possible to unset the dedicated columns via overrides, it would be necessary to make a distinction here between `nil` and the empty slice `[]backend.DedicatedColumn{}`. In this case the condition should be `cols != nil`

Reply:
Changed the tests in `overrides_test.go` to verify that the behaviour is as you describe.

Reply:
I just tested this and unfortunately this does not work: If `dedicated_columns` is set to `[]` in the overrides it takes the dedicated columns from the configuration. Therefore it is currently not possible to "unset" the dedicated columns.

In my opinion the following override configuration should remove all configured dedicated columns:
```yaml
overrides:
  dedicated_columns: []
```
But this isn't the case. If the line above is changed like this it works.
```go
    if cols := i.overrides.DedicatedColumns(i.instanceID); cols != nil {
```

Reply:
You're right. Changed 👍